doc/xroad_catalog_installation_guide.md

```diff
@@ -111,12 +101,32 @@ Add the X-Road repository’s signing key to the list of trusted keys:
 sudo rpm --import https://artifactory.niis.org/api/gpg/key/public
 ```
 
-Install the `xroad-confclient` module:
+## 2.4 Installation
+
+The installable software consists of `xroad-catalog-collector` and `xroad-catalog-lister` modules. Both are provided
+as RPM packages.
+
+```bash
+sudo yum install xroad-catalog-lister xroad-catalog-collector
+```
+
+Or alternatively:
 
 ```bash
-sudo yum install xroad-confclient
+rpm -i install xroad-catalog-lister xroad-catalog-collector
 ```
 
+Instructions on how to build the RPM packages using Docker can be found [here](../BUILD.md).
+
+## 2.4.1 Complete the Installation of X-Road Configuration Client
+
+**Note:** The configuration steps described in section 2.4.1 are needed only if the X-Road Catalog Lister is not installed on the
```

Comment:
I would change this line in the following way since otherwise it might be overlooked if the numbering changes in the future and cause confusion.

```suggestion
**Note:** The configuration steps described in this section are needed only if the X-Road Catalog Lister is not installed on the
```